core-bundle/src/Controller/BackendPreviewController.php

```diff
@@ -90,6 +90,6 @@ public function __invoke(Request $request): Response
             return new RedirectResponse($targetUrl);
         }
 
-        return new RedirectResponse('/');
+        return new RedirectResponse($this->previewScript ?: '/');
```

Comment:
This should probably use the request context's current root, right?

Reply:
See 0c321e4f585856f587828ceaafed5db71ddca534.